tailwind.config.ts

```diff
@@ -20,6 +20,26 @@ const config: Config = {
         'event-banner': '#5E18EB', // Event Banner Background
         socials: '#7BD4CB', // Social
       },
+      keyframes: {
+        float: {
```

Comment:
can you add a comment here explaing what this related too.

Additioally is this animation something that you could see us using again? If im not mistaken its pretty specific right? maybe we should call it like `hackathon_modal_float` or something.